### DISCUSSION TITLE
Creating new issue results in error

### DISCUSSION THREAD

**Discussion by Sebtho22** [szapp/patch-template#1] [Q&A] [answered]
Couldn't find any info on the error so i'm asking it here.

I tried to use the patch-template and create a new issue like stated in the manual.

Unfortunatly every try ends in an error stating "Repository information not available
Basic information about the repository is invalid. 
Please try again later"

Maybe i'm just missing something, but i can't seem to figure it out.
Any help is appreciated :)

**Comment by szapp** [comment, marked as answer]
Hi @Sebtho22  thanks for bringing that up! The error message is not very descriptive as I assumed this could never happen (i.e. be caught by another error message before).

A possible reason for this error is that your repository does not have a description. The description is used to initialize the comment in the resulting VDF and must not be empty. You can retry by opening a new issue in your repository after adding a repository description. Please report back if the error persists.

Thanks for your interesting and for taking the time to report the issue here.

**Reply by Sebtho22** [reply]
Thank you, somehow GitHub messed up the description change and i had to change it again. Never thought i'd be the DAU, but here i am... Anyway it fixed the issue and the script run without problems, Thank you!

**Reply by szapp** [reply]
Thanks for trying again and letting me know it works. No worries; that error message really isn't helpful. I should fix that when I find the time!

Cheers!

**Reply by szapp** [reply]
I finally got around to fixing the error message. When a description is missing, there will now be a more descriptive error.

---

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/user-attachments/assets/c7d60e5a-ff44-4dfc-9549-ebcf6ba6d2c7">
  <source media="(prefers-color-scheme: light)" srcset="https://github.com/user-attachments/assets/579258f3-6f59-4aa9-8f08-602c5c2e3f56">
  <img alt="" src="https://github.com/user-attachments/assets/579258f3-6f59-4aa9-8f08-602c5c2e3f56">
</picture>
